### ISSUE TITLE
Panelmenu: Issue with navigating by Tab through submenu items

### ISSUE THREAD

**Issue by lmormol** [primefaces/primeng#13630]
### Describe the bug

There is an issue related with navigating through submenu by Tabbing.

Here you can see incorrect behaviour:
https://drive.google.com/file/d/1ULrFyCfz33NiKysvXyCUsXaQGtd0CWKJ/view?usp=sharing

### Environment

Macos Apple M2
Ventura 13.5.1

### Angular version

16.2.3

### PrimeNG version

16.3.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.17.1

### Browser(s)

Latests: Chrome, Opera, Safari

### Steps to reproduce the behavior

Navigate by pressing [Tab] to first menu header named "File", open by clicking [Enter].
Then navigate by [Tab] to submenu item named "New", open by clicking [Enter].
Then navigate by [Tab] to "Delete" or "Export" and click.

### Expected behavior

There should be performed action related with currently focused element.
